src/jdk.incubator.jextract/share/classes/jdk/internal/clang/CLayouts.java

```diff
@@ -23,11 +23,11 @@
  *  questions.
  */
 
-package jdk.internal.clang.libclang;
+package jdk.internal.clang;
```

Comment:
This class should be removed. Other classes in the libclang package should use the C layouts that are generated in the main class header. Outside of the package, we should just use regular ValueLayouts (e.g. `ADDRESS`).